airbyte-integrations/connectors/source-salesforce/unit_tests/api_test.py

```diff
@@ -61,7 +64,7 @@ def test_bulk_stream_fallback_to_rest(mocker, requests_mock, stream_config, stre
     assert list(stream.read_records(sync_mode=SyncMode.full_refresh, stream_slice=stream_slices)) == rest_stream_records
 
 
-def test_stream_unsupported_by_bulk(stream_config, stream_api, caplog):
+def test_stream_unsupported_by_bulk(stream_config, stream_api):
```

Comment:
Not related to this test, but we should add tests for the `request_params` method. At the very least we should have a test that calls `request_params()` and verifies that even when a stream slice and a state that is later is passed in, we only generate the query string using the slice

Reply:
test added